app/controllers/admin/pages/sections_controller.rb

```diff
@@ -31,7 +31,8 @@ def update
     # Save edited page section details
     @section = PageSection.find( params[:id] )
 
-    if @section.update!( section_params )
+    _unused = @section.update( section_params )
+    if @section.valid?
```

Comment:
Apologies if you're already aware of this. `@section.update( section_params )` (no bang) returns the same boolean result as `@section.valid?`. 

This could be tweaked to, and is a little more conventional:

```diff
-_unused = @section.update( section_params )
-if @section.valid?
+if @section.update( section_params )
```

Reply:
This is how my code started out, but I think some combination of my tests and rubocop wasn't happy with it. I'll have another look at some point :)

Reply:
It could be that the original used `update!` which will raise an error for an invalid object, meaning the `else` branch would never run, whereas `update` (no bang) will return false for an invalid object.

Reply:
Just reverted this code and it works fine now, so ... I guess I was tired and doing something incredibly stupid the first time :D